crates/bitwarden/src/mobile/vault/client_ciphers.rs

```diff
@@ -12,9 +12,16 @@ pub struct ClientCiphers<'a> {
 }
 
 impl<'a> ClientCiphers<'a> {
-    pub async fn encrypt(&self, cipher_view: CipherView) -> Result<Cipher> {
+    pub async fn encrypt(&self, mut cipher_view: CipherView) -> Result<Cipher> {
         let enc = self.client.get_encryption_settings()?;
 
+        if cipher_view.key.is_none() && self.client.get_flags().enable_cipher_key_encryption {
```

Comment:
Can we add a comment reminding us to add this logic to the regular encryption logic when we remove this flag?